.eslintrc

```
@@ -0,0 +1,3 @@
+{
+  "extends": "standard"
```

```suggestion
  "extends": "standard",
  "root": true
```

Will need this to stop eslint picking up rules from parent directories